tools/abel/README.md

```diff
@@ -1,191 +1,233 @@
 
 ABEL has the following key features:
-* Convert a Google Sheet (in the ABEL Spreadsheet Template) to a building configuration YAML file
-  * Run Instance Validator (with optional telemetry validation) on the building configuration file
-* Convert a building configuration YAML file to a Google Sheet
 
-## Table of Contents
+* Create and edit a Google Sheet (in the ABEL Spreadsheet Template) and export it to a building configuration file for onboarding.
+ * Run Instance Validator (with optional telemetry validation) on the building configuration file during the conversion process to ensure there are no errors in the file prior to attempting onboarding.
+* Convert a building configuration file to a Google Sheet to make edits on.
+* Split a building configuration file to separate out a single namespace to make it easier to focus edits and onboarding processes to a single area as needed.
+
 
-* [Setup and Installation](#setup-and-installation)
+## User Guide Contents
+
+* [Getting Started](#getting-started)
+  *	[Set Up a Tooling Environment](#set-up-a-tooling-environment)
+  *	[Obtain a GCP OAuth Client Credential](#obtain-a-gcp-oauth-client-credential)
 * [Using ABEL](#using-abel)
-  * [Update Workflow](#update-workflow)
-  * [Initialization Workflow](#initialization-workflow)
-  * [Split Workflow](#split-workflow)
+  *	[General Use](#general-use)
+    *	[Update your Repository Clone and Activate your Virtual Environment](#update-your-repository-clone-and-activate-your-virtual-environment)
+    *	[ABEL Arguments](#abel-arguments)
+  * [Updating an Existing Building](#updating-an-existing-building)
+    *	[Create a Spreadsheet then Convert it to a Building Configuration File](#create-a-spreadsheet-then-convert-it-to-a-building-configuration-file)
+    *	[Generate a Building Configuration File from an Existing Spreadsheet](#generate-a-building-configuration-file-from-an-existing-spreadsheet)
+  * [Initializing a New Building](#initializing-a-new-building)
+  * [Splitting a Building Configuration File](#splitting-a-building-configuration-file)
 
-## Setup and Installation
 
-Before starting the setup and installation process, please ensure that the dependencies are met:
+## Getting Started
 
-1. You are running **Python 3.9** or higher
+ABEL is a Python tool that must be run in the command line. Before starting the setup and installation process outlined below, please ensure that the following dependencies are met:
 
-2. you have installed the [Google Cloud CLI](https://cloud.google.com/sdk/docs/install)
+* You are running [Python](http://go/swc) 3.9 or higher
+* You have installed the [Google Cloud CLI](https://cloud.google.com/sdk/docs/install)
```

Comment:
Technically you don't need the gcloud cli anymore since we are using OAuth keys for the google sheets API.